### ISSUE TITLE
Person ReID using Ranked List Loss, IMAE, OSM and CAA, ....

### ISSUE THREAD

**Issue by XinshaoAmosWang** [michuanhaohao/reid-strong-baseline#68]
So glad to hear from you, looking forward to discussion and sharing with you: 

1. ReID using RLL: https://github.com/Qidian213/Ranked_Person_ReID

2. ReID results using RLL will be shown: https://github.com/XinshaoAmosWang/Ranked-List-Loss-for-DML

3. ReID using OSM and CAA: https://github.com/ppriyank/-Online-Soft-Mining-and-Class-Aware-Attention-Pytorch
https://arxiv.org/abs/1811.01459 

4. ReID on MARS using IMAE: https://github.com/XinshaoAmosWang/Improving-Mean-Absolute-Error-against-CCE

Related Papers: 
OSM and CAA: https://arxiv.org/abs/1811.01459 (metric learning & classification)
RLL: https://arxiv.org/abs/1903.03238 (metric learning)
IMAE: https://arxiv.org/pdf/1903.12141.pdf (classification)

**Comment by michuanhaohao** [comment]
Thank you very much! I will keep this issue open. Everyone can discuss it.

**Comment by XinshaoAmosWang** [comment]
I appreciate it greatly @michuanhaohao

**Comment by bochuanwu** [comment]
i use osm on my loss function but it seems not a good loss function for me， maybe i misunderstand the loss fuction.
![image](https://user-images.githubusercontent.com/35745976/69842621-ea1b1e80-129e-11ea-8877-49a864a64510.png)

**Comment by XinshaoAmosWang** [comment]
@bochuanwu , 

I will release the code for your reference. Thanks for your interest in the work.

**Comment by XinshaoAmosWang** [comment]
Good news!

I released the code of `Deep Metric Learning by Online Soft Mining and Class-Aware Attention, AAAI 2019 Oral' 
https://github.com/XinshaoAmosWang/OSM_CAA_WeightedContrastiveLoss

**Comment by bochuanwu** [comment]
> Good news!
> 
> We released the code of `Deep Metric Learning by Online Soft Mining and Class-Aware Attention, AAAI 2019 Oral'
> https://github.com/XinshaoAmosWang/OSM_CAA_WeightedContrastiveLoss
> 
> Thank you so much for your interest.
> Please feel free to provide us your feedbacks. Thanks.

thank you for your reply， i will check the code later.
